### PR TITLE
New node types

### PR DESCRIPTION
**Pull request by maximusunc**
Adding new node types with colors, upgrading npm packages to stop the emails.

### CODE REVIEW

File: src/components/util/colorUtils.js [hunk 1]
```diff
@@ -11,20 +11,26 @@ const conceptColorMap = {
...
   disease_or_phenotypic_feature: '#fbb4ae', // Red, same as disease
   drug: '#8787ff', // Purply blue, same as chemical_substance
   protein: '#ccebc5', // Green like gene
+  environmental_feature: '#8a9a5b', // Moss Green
+  food: '#ffa343', // Neon Carrot
   gene: '#ccebc5', // Green
   gene_family: '#68c357', // Darker Green
   genetic_condition: '#ffffcc', // Yellow
   gross_anatomical_structure: '#f1ebe0', // Lighter brown to go with anatomical_entity
+  life_stage: '#fe4164', // Neon Fuchsia
```
Comment (kennethmorton) [inline src/components/util/colorUtils.js]: This is a vile, disgusting color. No need to change it though.

File: manager/api/misc_api.py [hunk 2]
```diff
@@ -281,6 +281,10 @@ def get(self, id1):
         """
```
Comment (kennethmorton) [inline manager/api/misc_api.py]: Why all these changes now? They seem fine, but more like they should have been committed a long time ago?

**Comment by kennethmorton** [comment]
Minor comments but otherwise, thumbs up

File: manager/api/misc_api.py (hunk 2 above)
Comment (maximusunc) [reply manager/api/misc_api.py]: Yeah, seems like I was sitting on them since the hackathon.

**Pull request merged**